### PR TITLE
Remove docs refs

### PR DESCRIPTION
**Pull request by dovholuknf**

### CODE REVIEW

**Review by gberl002 — APPROVED** [review]

**Review by qrkourier — CHANGES_REQUESTED** [review]
I ran the crawler script on the preview and found one issue.

```bash
./check-links/crawl-for-broken-links.sh https://ziti-doc-git-remove-docs-refs-openziti.vercel.app/
```

```
/docs/guides/securing-apis/aks-api-with-nginx-ziti-module
        id #enrolling- not found        /docs/reference/tunnelers/windows#enrolling-
```

File: docusaurus/docs/guides/securing-apis/aks-api-with-nginx-ziti-module.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ client name = `client-nginx` with `Attribute`: `#clients`,  server module name =
 
 Download jwt files and enroll identities. 
 
-- Windows ZDE Identity can be enrolled by following this [enrolling process](https://docs.openziti.io/docs/reference/tunnelers/windows#enrolling)
+- Windows ZDE Identity can be enrolled by following this [enrolling process](../../../docs/reference/tunnelers/02-windows.md#enrolling-)
```
Comment (qrkourier) [inline docusaurus/docs/guides/securing-apis/aks-api-with-nginx-ziti-module.md]:
```suggestion
- Windows ZDE Identity can be enrolled by following this [enrolling process](../../../docs/reference/tunnelers/02-windows.md#enrolling)
```

Comment (qrkourier) [reply docusaurus/docs/guides/securing-apis/aks-api-with-nginx-ziti-module.md]: fixes a broken fragment link

**Review by qrkourier — DISMISSED** [review]

File: docusaurus/docs/guides/securing-apis/aks-api-with-nginx-ziti-module.md (hunk 1 above)
Comment (dovholuknf) [reply docusaurus/docs/guides/securing-apis/aks-api-with-nginx-ziti-module.md]: golang put that there and i didn't bother to verify it. I tried it out and I see that it works fine but the `-` seems to 'hide the title': https://ziti-doc-git-remove-docs-refs-openziti.vercel.app/docs/reference/tunnelers/windows#enrolling-

so i prefer without the `-`

**Pull request merged**